2022-round-2/abrhamgg/nx_dev_test_output.txt

```diff
@@ -0,0 +1 @@
+PYTHONPATH=. pytest networkx
```

Comment:
Could you paste the output of the test suite?
Thanks!

Reply:
done